src/OrbitGl/ThreadTrack.h

```diff
@@ -80,6 +80,8 @@ class ThreadTrack final : public TimerTrack {
                                    PickingMode picking_mode, float z_offset);
   void UpdateMinMaxTimestamps();
 
+  int32_t thread_id_;
```

Comment:
nit: int64_t while you are here?

Reply:
I don't understand your comment, I used that because it was int32 before. I'm changing because I suppose we wanted to change it?